.env

```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=cbvjsbdfhdsjkfnsthis
```

Comment:
you should not push `.env` on github

Reply:
Is there another way to pass `REACT_APP_JWT_SECRET` in deployment?

Reply:
either configure it with [netlify.toml](https://docs.netlify.com/configure-builds/file-based-configuration/) file or directly give the env values on netlify in build in deploy settings.

Reply:
Ok got it. I will delete .env and add it to environment variables on netlify settings.  Thank you @rohanmathur91 for this.